docs/swagger-ui.html

```diff
@@ -0,0 +1,60 @@
+<!-- HTML for static distribution bundle build -->
```

Comment:
Perhaps we can deprecate Swagger UI?

Reply:
While believe that Swagger UI are too complex for API documentation, it does have a two important differences - widespread/enterprise adoption and ability to execute API examples (`Try It`)

So I would still have verification that our Open API specification works in SwaggerUI too.